meson.build

```diff
@@ -529,6 +529,9 @@ else
         conf.set10('HAVE_GETRANDOM', have)
 endif
 
+glibc_uses_libidn2 = cc.get_define('NI_IDN_USE_STD3_ASCII_RULES', prefix : '''#include <netdb.h>''', args : '-D_GNU_SOURCE') != '128'
```

Comment:
Hmm, that seems very magicky and brittle. Maybe we can make a more direct test?

Reply:
Do you have any idea for that? Unfortunately the patch dows not provide any flag whether glibc has the patch or not, if I understand correctly. Of course, if there is, we should use it.